Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -293,3 +293,5 @@ spec:
        sum(rate(ceph_rgw_put[2m])) # promethues query used for autoscaling
      threshold: "90"
 ```
+
+P.S During reconciliation of `cephobjectstore`, the Rook Operator will reset the replica count for RGW which was set by horizontal pod scaler. And when it identifies horizontal pod scaler revert the change again. This is briefly discussed (here)[https://github.com/rook/rook/issues/10001]
```

Comment:
Let's use the new callout format here. @travisn do you think 'note' or 'warning' here?

```suggestion
!!! warning
    During reconciliation of `cephobjectstore`, the Rook Operator will reset the replica count for RGW which was set by horizontal pod scaler. And when it identifies horizontal pod scaler revert the change again. This is briefly discussed (here)[https://github.com/rook/rook/issues/10001]
```